stream_alert/rule_processor/handler.py

```diff
@@ -68,11 +69,18 @@ def run(self, event):
             else:
                 LOGGER.info('Unsupported service: %s', payload.service)
```

Comment:
@jacknagz , @mime-frame I will change this to error but please keep in mind a LOT of this code is changing in the next upcoming PR. Maybe don't nit about lines I haven't actually changed right now ;)